site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,159 +3,74 @@ layout: 'layouts/doc-post.njk'
 title: 'Match patterns'
 seoTitle: 'Chrome Extensions match patterns'
 date: 2012-09-18
-updated: 2023-07-28
+updated: 2023-08-31
 description: Understanding URL match patterns in Chrome extensions.
 ---
 
-## Overview {: #overview }
-
-Extensions use match patterns to specify a group of URLs. They are mainly used to declare [host permissions][host-permissions] along with other permissions to use certain Chrome APIs or when injecting [content scripts][content-scripts]. Extensions also use match patterns in:
-
-- [`"web_accessible_resources.matches"`][war]
-- [`"externally_connectable.matches"`][ext-connect]
-
-## What is a match pattern? {: #what }
-
-A match pattern is a URL that begins with a permitted scheme, separated by a `://`, followed by a host and path. It can contain wildcard (`*`) characters. Most match patterns are composed of three parts:
+A match pattern is a URL with the following structure, used to specify a group of URLs:
 
 ```text
 &lt;scheme&gt;://&lt;host&gt;/&lt;path&gt;
 ```
 
-Each part can use wildcards `*`. Below is a detailed description:
+**scheme**: Must be one of `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
+  - For information on injecting content scripts into unsupported schemes, such as `about:` and `data:`, see [Injecting in related frames][cs-frames].
+  - The scheme must be separated from the rest of the pattern using a double slash (`//`).
 
-- **scheme**: Must include a valid scheme: `'http'`, `'https'`, `'file'`, `'ftp'`, or `'urn'`. The wildcard `*` only matches `http` or `https`.
+**host**: A hostname (`www.example.com`). A `*` before the hostname to match subdomains (`*.example.com`), or just a wildcard `*`.
+  - If you use a wildcard in the host pattern, it must be the first or only character, and it must be followed by a period (`.`) or forward slash (`/`).
```

Comment:
Is this right? Later in the same article we talk about things like `/foo*`.

Reply:
I don't know. @AmySteam?

Reply:
@AmySteam

Reply:
The  `/foo*` example below is for the **_path_**, not the **_host_**. So this is correct.